lwconfig/config.go

```diff
@@ -66,12 +66,24 @@ func (p *Profile) Verify() error {
 	if p.Account == "" {
 		return errors.New("account missing")
 	}
+
 	if p.ApiKey == "" {
 		return errors.New("api_key missing")
 	}
+
+	if len(p.ApiKey) < 55 {
+		return errors.New("api_key must have more than 55 characters")
+	}
```

Comment:
Out of interest, where are these values pulled from?

Reply:
https://github.com/lacework/go-sdk/blob/main/cli/cmd/configure.go#L274

Would be better if this validation was in a single location.

Reply:
Got it. Yeah it would be nice if this could just be validated in this location and updated/removed in configure.go